### DISCUSSION TITLE
In Megane 4, the lane tracking system is disabled with the Automatic High Beam check signal that appears when the battery is low.

### DISCUSSION THREAD

**Discussion by KasifG** [cedricp/ddt4all#984] [Q&A]
Hello everyone, 
How can I use the ddt4all or pyRen program to reactivate the lane tracking system, which is disabled with the Automatic High Beam signal that appears when the battery is low in Megane 4? 
Or do I need to go to a service other than these programs?
Thanks in advance

**Comment by KasifG** [comment]
When the battery got weak, I ordered it and replaced it myself in my garage. However, the signals I mentioned above were not deleted and both features are not working at the moment. I would be very happy if you could help me on how to enable it. I don't want to go to the service.

**Reply by KasifG** [reply]
Fault Signal was “No Camera Multiplex Signal”

**Comment by KarelSvo** [comment]
Already tried?
Start the engine, turn the steering wheel a few times to the two end points

**Reply by KasifG** [reply]
İ tried right now but it did not work.
